### PR TITLE
Update standalone schedule name uniqueness combining it with unified …

### PR DESCRIPTION
**Pull request by ghost**
…job template.

Signed-off-by: Vismay Golwala <vgolwala@redhat.com>

##### SUMMARY
Existing schedule interface only allows unique name across all the schedules which doesn't sound great as there is no way of knowing which name has been used. So in order to address this issue, we can set combined uniqueness of schedule ```name``` and the foreign key (```unified_job_template```). So we would reject the schedule name only when there exists one with the same name using the same job template.

##### ISSUE TYPE
 - Bugfix Pull Request
 
##### COMPONENT NAME
- API
- UI

##### AWX VERSION

```
awx: 3.0.0
```

### CODE REVIEW

File: awx/main/migrations/0059_auto_20190207_2022.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2019-02-07 20:22
```
Comment (wenottingham) [inline awx/main/migrations/0059_auto_20190207_2022.py]: Please rename this file to something descriptive (see other migrations).

File: awx/main/migrations/0059_auto_20190207_2022.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+from __future__ import unicode_literals
```
Comment (kdelee) [inline awx/main/migrations/0059_auto_20190207_2022.py]: :thinking: what is this needed for? :)

Comment (ghost) [reply awx/main/migrations/0059_auto_20190207_2022.py]: I thought we were managing both Py 2 and 3. Removed now. Thanks!

**Review by ryanpetrello — APPROVED** [review]

**Review by AlanCoding — APPROVED** [review]
change of the base class makes sense to me to alter the name field.

File: awx/main/migrations/0059_v350_update_schedule_name_uniqueness.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2019-02-07 20:22
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0058_v350_remove_limit_limit'),
```
Comment (ryanpetrello) [inline awx/main/migrations/0059_v350_update_schedule_name_uniqueness.py]: This is outdated now due to https://github.com/ansible/awx/pull/3225/

**Pull request merged**